### PR TITLE
adds product details page and connect with redux

### PR DESCRIPTION
**Pull request by shubhamnarang11**

### CODE REVIEW

File: src/components/Carousel/Carousel.scss [hunk 1]
```diff
@@ -58,14 +65,18 @@
...
 
 .left-arrow {
-  left: 24px;
+  left: 5px;
 }
 
 .right-arrow {
-  right: 24px;
+  right: 5px;
 }
```
Comment (sumantasau) [inline src/components/Carousel/Carousel.scss:80]: As per your suggestion not use px, please convert it rem

File: src/components/Navbar/Navbar.scss [hunk 2]
```diff
@@ -58,6 +58,17 @@
...
+  #cart-total {
+    font-size: 12px;
+    background: #ff0000;
+    color: #fff;
+    padding: 1px 4px;
+    border-radius: 50%;
+    position: absolute;
+    margin-right: -10px;
+    margin-top: -5px;
+  }
```
Comment (sumantasau) [inline src/components/Navbar/Navbar.scss:71]: Here also convert px to rem

File: src/components/Carousel/Carousel.scss (hunk 1 above)
Comment (shubhamnarang11) [reply src/components/Carousel/Carousel.scss:80]: Done

File: src/components/Navbar/Navbar.scss (hunk 2 above)
Comment (shubhamnarang11) [reply src/components/Navbar/Navbar.scss:71]: Done

**Pull request merged**